samples/kvsWebRTCClientMaster.c

```diff
@@ -240,10 +239,16 @@ PVOID sendVideoPackets(PVOID args)
             status = writeFrame(pSampleConfiguration->sampleStreamingSessionList[i]->pVideoRtcRtpTransceiver, &frame);
             encoderStats.encodeTimeMsec = 4; // update encode time to an arbitrary number to demonstrate stats update
             updateEncoderStats(pSampleConfiguration->sampleStreamingSessionList[i]->pVideoRtcRtpTransceiver, &encoderStats);
-            if (status != STATUS_SUCCESS) {
+            if (status != STATUS_SUCCESS && status != STATUS_SRTP_NOT_READY_YET) {
```

Comment:
This looks wrong. How about

if (status != STATUS_SRTP_NOT_READY_YET) {
  if (STATUS_FAILED(status)) { blah} else if (firstFrame) {print}
}

Reply:
Reason for not doing that is to avoid multiple if levels unnecessarily. That is all. But, I have made the change